aten/src/ATen/native/native_functions.yaml

```diff
@@ -760,6 +760,10 @@
 
 - func: conv2d(Tensor input, Tensor weight, Tensor? bias=None, int[2] stride=1, int[2] padding=0, int[2] dilation=1, int groups=1) -> Tensor
 
+- func: conv2d_prepack(Tensor weight, Tensor? bias=None, int[2] stride=1, int[2] padding=0, int[2] dilation=1, int groups=1, float output_min=-std::numeric_limits<float>::infinity(), float output_max=std::numeric_limits<float>::infinity()) -> Tensor
+
+- func: conv2d_run(Tensor context, Tensor input) -> Tensor
```

Comment:
What do you want call this?

Reply:
I'd probably call this conv2d_prepacked_run.  We can change this after this lands, though, as long as we haven't exposed a frontend yet.

Reply:
These are not meant to be directly used by users, right? They should be prefixed with underscore in that case

Reply:
if the names aren't public and the implementations are xnnpack specific, maybe just call them something like
`_conv2d_xnnpack_run`?

Reply:
I was under the impression that these are supposed to be generic, say for sparse NN stuff as well - not sure.  Wasn't moving these to native_functions partly motivated so as to use the dispatch machinery and consequently multiple backend support?  I am not sure really.  I'll let @dzhulgakov answer.  P.S. I'll mark the functions with underscore.

Reply:
well, we (mostly) support the dispatch machinery outside of native functions.  But to have the same operator dispatch to different backends you need different dispatch keys.  It's not obvious to me how that maps here.

Reply:
@dzhulgakov If you don't predict anyone from the OSS world calling `conv2d_packed` directly, it's better to be conservative and make it private first, and then fix it later. You just have  lot more rope that way.

Reply:
right, it seems pretty clear this is not the final API (see e.g. the discussion about not returning Tensors), so putting it in the public API (not using an underscore) doesn't seem like a good idea.